File: 2581/2581.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    2581.py                                            :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/09/29 23:45:06 by jihoolee          #+#    #+#              #
#    Updated: 2021/09/29 23:52:14 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

start = int(input())
end = int(input())

def is_prime(num: int):
	if (num == 1):
		return (False)
	for i in range(2, num):
		if (num % i == 0):
			return (False)
	return (True)

prime_num_lst = [num for num in range(start, end + 1) if is_prime(num)]
if (prime_num_lst == []):
	print(-1)
else:
	print(sum(prime_num_lst))
	print(min(prime_num_lst))

File: 2581/2581.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   2581.cpp                                           :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/09/29 23:38:20 by jihoolee          #+#    #+#             */
/*   Updated: 2021/09/29 23:44:37 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>

using namespace std;

bool	is_prime(int num)
{
	if (num == 1)
		return (false);
	for (int i = 2; i < num; i++)
		if (num % i == 0)
			return (false);
	return (true);
}

int	main(void)
{
	int	start, end;
	int	sum = 0;
	int	min = -1;

	cin >> start >> end;
	for (int i = start; i <= end; i++)
	{
		if (is_prime(i))
		{
			sum += i;
			if (min == -1)
				min = i;
		}
	}
	if (sum > 0)
		cout << sum << endl;
	cout << min << endl;
	return (0);
}
